src/main/java/javax/jmdns/impl/DNSCache.java

```diff
@@ -218,10 +218,10 @@ public boolean addDNSEntry(DNSEntry dnsEntry) {
             if (entryList == null) {
                 entryList = new ArrayList<>(3);
                 entryList.add(dnsEntry);
```

Comment:
```suggestion
```

Reply:
> Yes, corrected.

Where exactly? I still see it...

Reply:
Number of commits in the pull request changed from 3 back to 2 again. I've accepted yours